docs/source/additional_features.rst

```diff
@@ -348,13 +349,14 @@ Extended Callable types
    This feature is deprecated.  You can use
    :ref:`callback protocols <callback_protocols>` as a replacement.
 
-As an experimental mypy extension, you can specify ``Callable`` types
-that support keyword arguments, optional arguments, and more.  When
-you specify the arguments of a Callable, you can choose to supply just
-the type of a nameless positional argument, or an "argument specifier"
-representing a more complicated form of argument.  This allows one to
-more closely emulate the full range of possibilities given by the
-``def`` statement in Python.
+As an experimental mypy extension, you can specify
+:py:data:`~typing.Callable` types that support keyword arguments,
+optional arguments, and more. When you specify the arguments of a
+``Callable``, you can choose to supply just the type of a nameless
```

Comment:
As an example of the KISS principle I mentioned, this second occurrence of `Callable` could easily be linked as well, even though it's in the same paragraph as the first.